tests/Behat/Resources/suites/ui/product/managing_product_variants.yaml

```diff
@@ -0,0 +1,34 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        api_managing_product_variants:
```

Comment:
This file should probably live in the `tests/Behat/Resources/suites/**api**/product` directory